patches/0118-libavfilter-dnn-dnn_backend_torch-add-multi-device-s.patch

```diff
@@ -0,0 +1,301 @@
+From d43f502042abb5e1974e7beb27a5f8fb3b326c1a Mon Sep 17 00:00:00 2001
+From: Wenbin Chen <wenbin.chen@intel.com>
+Date: Mon, 20 Nov 2023 14:32:26 +0800
+Subject: [PATCH 2/2] libavfilter/dnn/dnn_backend_torch: add multi-device
+ support
+
+Now ffmpeg-libtoch can be run on multi-device at same time. When set device
+user need to input all devices splited by '&' and also set async=1. The
+inference tasks will be distributed to idle device dynamically.
```

Comment:
could you provide an example in the log ?

Reply:
done, thanks.